source/mentoring-service/source/events-speakers.md

```diff
@@ -2,7 +2,7 @@
 
 ![Luke Killick](/source/images/speakers/Luke-Killick.png)
 
-Luke Killick
+## Luke Killick
 =============
```

Comment:
@naja-ahmed this line now appear wrongly on the page, please check before the PR.